aiida/backends/djsite/utils.py

```diff
@@ -162,16 +162,23 @@ def check_schema_version(profile_name):
         set_db_schema_version(code_schema_version)
         db_schema_version = get_db_schema_version()
 
-    if code_schema_version != db_schema_version:
+    if code_schema_version > db_schema_version:
```

Comment:
This code also exists for SqlAlchemy, maybe you want to update it there as well. Although this is not as easy as there you cannot simply compare revisions by their value. Anyway this code is touched heavily by PR #3582

Reply:
Ok, so how would one do it?
If there's no good way to do it, I guess we can leave the sqlalchemy version untouched.

Reply:
It's fine for now, but these changes will probably become irrelevant with PR #3582 . But we can merge this now.